dev/ci/scenarios.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export DISPLAY=:99.0
+if [ "${TRAVIS_OS_NAME}" == "osx" ]; then
+  ( sudo Xvfb :99 -ac -screen 0 1024x768x8; echo ok )&
+else
+  sh -e /etc/init.d/xvfb start;
+fi
+sleep 3 # give xvfb some time to start
```

Comment:
Maybe we could extract and source the xvfb hack?

Reply:
Actually I did that before but it seems it needs to run in the same shell as the following commands to work.

Reply:
But maybe that's possible to do with something like `("path/to/script")`

Reply:
I thought `source some/script.sh` could work, but I may be wrong...

Reply:
you were not ;-)